src/service/data_products/samples.py

```diff
@@ -164,18 +165,19 @@ class Samples(BaseModel):
 @_ROUTER.get(
     "/",
     response_model=SamplesTable,
-    description="Get the corresponding sample attributes for each data unit in the collection, "
-        + "which may differ from collection to collection. Note that data units may share "
-        + "samples - if so the sample information is duplicated in the table for each data unit."
-        + "\n\n "
+    description="Get the sample attributes for the data units in the collection, "
+        + "which may differ from collection to collection.\n\n"
         + "Authentication is not required unless submitting a match ID or overriding the load "
         + "version; in the latter case service administration permissions are required.\n\n"
-        + "When creating selections from samples, use the "
-        + f"`{names.FLD_KBASE_ID}` field values as input.")
+    # TODO SAMPLES - how should we support creating a selection from samples?
+)
 async def get_samples(
     r: Request,
     collection_id: str = PATH_VALIDATOR_COLLECTION_ID,
-    sort_on: common_models.QUERY_VALIDATOR_SORT_ON = names.FLD_KBASE_ID,
+    sort_on: Annotated[str, Query(
+        example=names.FLD_KB_SAMPLE_ID,
+        description="The field to sort on."
+    )] = names.FLD_KB_SAMPLE_ID,
```

Comment:
Do you think we should sort on `genome_count` by default?

Reply:
Actually now I think about it we should search on the display name. I forgot about it because it's not in the flattened data, but that's because we haven't redownloaded the genomes because podman is still messed up

Reply:
👍 I do have a script to manually update the sample.prepare file. I can ensure the data has display_name when we deploy this change.